### ISSUE TITLE
Error:  Got 'if-feature' - ietf-subscribed-notifications@2019-09-09

### ISSUE THREAD

**Issue by sprinto** [vlvassilev/yuma123#110]
Hi,
I'm testing yangcli 2.12-0 with tail-f ConfD Netconf server (confd-basic-7.5.1). When yangcli connects to the server it has problem with ietf-subscribed-notifications@2019-09-09.yang:

```
yangcli:connect> 
val->res is NO_ERR.

yangcli: Starting NETCONF session for admin on host.docker.internal
Warning: skipping unknown parameter 'content-id=be923b87f292514d1f71a758254088f8' in capability URI 'urn:ietf:params:netconf:capability:yang-library:1.1?revision=2019-01-04&content-id=be923b87f292514d1f71a758254088f8'
Warning: 'module' parameter missing from possible capability URI 'urn:ietf:params:netconf:capability:yang-library:1.1?revision=2019-01-04&content-id=be923b87f292514d1f71a758254088f8'

NETCONF session established for admin on host.docker.internal

...

Getting yang-library module set  ...
Current module-set-id: be923b87f292514d1f71a758254088f8.


Checking Server Modules...

Error:  Got 'if-feature', Expected: identity sub-statement
ietf-subscribed-notifications@2019-09-09.yang:223.5: error(246): wrong token value

Error:  Got 'if-feature', Expected: identity sub-statement
ietf-subscribed-notifications@2019-09-09.yang:277.5: error(246): wrong token value

...
```
Looks like the same problem as  #90 but ietf-subscribed-notifications is not a draft?

**Comment by lightside-instruments** [comment]
Correct. I will fix that ASAP.

**Comment by vlvassilev** [comment]
Correct. I will fix that ASAP.

**Comment by sprinto** [comment]
Great, how do I enjoy the fix? Will it be a new release?

**Comment by vlvassilev** [comment]
You can compile the version in trunk. Just verified yangcli connects OK (with some warnings) to the latest confd 7.5.1. The issue you report was fixed here 52ffc218af1e718ddb1ed88f861db41dd414c8ef.

`yangcli --server=localhost --ncport=2022 --user=admin --password=admin`

We will make a 2.13 release soon.
